cmd/snap/cmd_delete_key_test.go

```diff
@@ -21,12 +21,32 @@ package main_test
 
 import (
 	"encoding/json"
+	"os"
 
 	. "gopkg.in/check.v1"
 
 	snap "github.com/snapcore/snapd/cmd/snap"
+	"github.com/snapcore/snapd/testutil"
 )
 
+// XXX: share this helper with signtool tests?
```

Comment:
(nitpick) maybe having a `sigtool/sigtooltest` package but definitely fine for a followup